peer/peer.go

```diff
@@ -14,23 +14,25 @@ import (
 	swarm "github.com/libp2p/go-libp2p-swarm"
 	bhost "github.com/libp2p/go-libp2p/p2p/host/basic"
 	ma "github.com/multiformats/go-multiaddr"
+	log "github.com/sirupsen/logrus"
+	msg "github.com/ubclaunchpad/cumulus/message"
+	sn "github.com/ubclaunchpad/cumulus/subnet"
 )
 
 const (
 	// DefaultPort is the TCP port hosts will communicate over if none is
 	// provided
 	DefaultPort = 8765
-
 	// CumulusProtocol is the name of the protocol peers communicate over
 	CumulusProtocol = "/cumulus/0.0.1"
-
 	// DefaultIP is the IP address new hosts will use if none if provided
 	DefaultIP = "127.0.0.1"
 )
 
 // Peer is a cumulus Peer composed of a host
 type Peer struct {
 	host.Host
+	subnet sn.Subnet
```

Comment:
I _thiiiiink_ we're using capital letters, see https://github.com/ubclaunchpad/cumulus/pull/23/files `Subnet`